### ISSUE TITLE
Resolve `svelte-ignore a11y-role-has-required-aria-props`

### ISSUE THREAD

**Issue by metonym** [metonym/svelte-typeahead#72]
Follow up to #71 

The `svelte-ignore a11y-role-has-required-aria-props` warning emitted by Svelte@3.52 is temporarily ignored in #71 but should be addressed.
